public/data/aladdin.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Inaolaji Taofeek",
+  "type": "personal",
+  "bio": "Full Stack Software Engineer | Open Source Enthusiast!",
+  "avatar": "https://github.com/aladdin4u.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/aladdin4u",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/devaladdin",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I recommend capitalizing the `i` here. What do you think? 🙂
```suggestion
      "name": "Follow me on LinkedIn",
```